po/es.po

```diff
@@ -1964,8 +1964,8 @@ msgid ""
 "```"
 msgstr ""
 "```shell\n"
-"$ cargo new exercise\n"
-"     Created binary (application) `exercise` package\n"
+"$ cargo new ejercicio\n"
```

Comment:
don't translate this or you'll be forced to translate all the cargo output below that shows the original src folder.